src/test/java/core/basesyntax/service/impl/ParseServiceImplTest.java

```diff
@@ -0,0 +1,67 @@
+package core.basesyntax.service.impl;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.exceptions.FruitShopException;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.ParseService;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+class ParseServiceImplTest {
+    private static final int BALANCE_INDEX = 0;
+    private static final int SUPPLY_INDEX = 1;
+    private static final int PURCHASE_INDEX = 2;
+    private static final int RETURN_INDEX = 3;
+    private static final String FRUIT = "banana";
+    private static final int QUANTITY = 100;
+    private static ParseService parseService;
+    private static List<String> inputData;
+    private static List<String> emptyList;
+    private static List<FruitTransaction> parseData;
```

Comment:
```suggestion
    private static ParseService parseService;
```
you don't really need these variables. Create them and use them in the test methods you need. Likewise in other test classes